time stamp

# latest tray and object combination 
uint32 tray_obj_combination

# R_tray: 	1 = Red object in Red tray 
# 		2 = Green object in Red tray 
#		4 = Blue  object in Red tray 
# G_tray: 	8 = Red object in Green tray 
#		... 
#		...
# B_tray: 	64 = Red object in Blue tray 
#		... 
#		... 	


# latest object color
uint32 current_object 

# 1 = Red 
# 2 = Green 
# 3 = Blue 
---

bool success
